indico/modules/events/registration/templates/management/_sortable_list.html

```diff
@@ -0,0 +1,28 @@
+{% macro sortable_items(items, actions_macro) -%}
+    {% for item in items %}
+        <li class="i-label sortable-item" id="{{ item.id }}">
+            <div class="actions">
+                <div class="toggle-enabled"></div>
+            </div>
+            <div class="title">{{ item.title }}</div>
+            <div class="actions">
+                <div class="move-prev"></div>
+                <div class="move-next"></div>
+                {% if not actions_macro is none %}
```

Comment:
isn't it possible to do `{% if actions_macro %}`? Could it have empty string as a value for instance?